data/json/items/book/dodge.json

```
@@ -55,6 +146,59 @@
     "category": "manuals",
     "name": { "str": "Treasury of Legends about Western Dancing", "str_pl": "copies of Western Dancing" },
     "description": "Written by Emanuel Nogueira, a constabulario and historian of Nuevo Laredo, this massive book details the movements and cultural legacies of a variety of North American folk dances.",
+    "variants": [
+      {
+        "id": "book_nonf_hard_dodge_tlwd",
+        "name": { "str": "Treasury of Legends about Western Dancing", "str_pl": "copies of Treasury of Legends about Western Dancing" },
+        "description": "Written by Emanuel Nogueira, a constabulario and historian of Nuevo Laredo, this massive book details the movements and cultural legacies of a variety of North American folk dances.",
+        "weight": 1
+      },
+      {
+        "id": "book_nonf_hard_dodge_tlwd_1",
+        "name": { "str": "A Guide To Making Yourself A Hard Target", "str_pl": "copies of A Guide To Making Yourself A Hard Target" },
+        "description": "Self defense fundamentals, this book describe techniques and move set you need to train to not be a victim.",
+        "//": "979-8620742561",
+        "weight": 1
+      },
+      {
+        "id": "book_nonf_hard_dodge_tlwd_2",
+        "name": {
+          "str": "The Ultimate Guide to Unarmed Self Defense",
+          "str_pl": "copies of The Ultimate Guide to Unarmed Self Defense"
+        },
+        "description": "The book contain lot of of info about awareness, prevention and defense in unarmed fight.",
+        "//": "978-1497530201",
+        "weight": 1
+      },
+      {
+        "id": "book_nonf_hard_dodge_tlwd_3",
+        "name": { "str": "Knife Self-Defense for Combat", "str_pl": "copies of Knife Self-Defense for Combat" },
+        "description": "Small manual about skills you need to obtain to defend yourself in dangerous street fight.",
+        "//": "978-0897500227",
+        "weight": 1
+      },
+      {
+        "id": "book_nonf_hard_dodge_tlwd_4",
+        "name": { "str": "Invincible", "str_pl": "copies of Invincible" },
+        "description": "Dark book with some barbarian on the cover, this book talks about what and how to obtain skills for the streets, battlefields and… playing fields?",
```

```suggestion
        "description": "A dark book with a barbarian on the cover, this guide talks about what and how to obtain skills for the streets, battlefields and… playing fields?",
```
I was also confused by the book's subtitle 😛